docker/base/os/debian/Dockerfile.buster

```diff
@@ -2,7 +2,7 @@
 
 FROM debian:buster
```

Comment:
Why build for both `bullseye` and `buster`?

Reply:
many open source projects does it to test with (usually) last 2 OS major versions (if possible LTS, or at least latest LTS and latest non-LTS); but I agree, maybe only latest here could be enough